client/starwhale/cli/assistance/broker.py

```diff
@@ -0,0 +1,288 @@
+import threading
+from typing import Any, Dict, List, Optional
+from datetime import datetime
+
+from fastapi import Body, FastAPI, Request, Response, HTTPException
+from pydantic import BaseSettings
```

Comment:
add `pydantic` dependency into setup.py ?

Reply:
broker is a separate service and is not expected to be deployed by users. It should not be added to swcli's depedencies.